src/components/ManageAccountsDrawer/ManageAccountsDrawer.tsx

```diff
@@ -19,11 +22,12 @@ export const ManageAccountsDrawer = ({
   onClose,
   chainId: parentSelectedChainId,
 }: ManageAccountsDrawerProps) => {
+  const wallet = useWallet().state.wallet
   const [step, setStep] = useState<ManageAccountsStep>('selectChain')
   const [selectedChainId, setSelectedChainId] = useState<ChainId | null>(null)
 
   // TODO: Implement Ledger specific logic
```

Comment:
We can probably remove this guy already?

Reply:
Right you are! https://github.com/shapeshift/web/pull/6779/commits/7c2c88eed41d6fbafe78a07a6f1add6c2ead8b33